src/components/AirbaseNG/AirbaseNG.tsx

```diff
@@ -1,165 +1,171 @@
-import { Button, Stack, Switch, TextInput } from "@mantine/core";
+import { Button, Stack, Switch, TextInput, Tooltip } from "@mantine/core";
 import { useForm } from "@mantine/form";
 import { useCallback, useState, useEffect } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
 import ConsoleWrapper from "../ConsoleWrapper/ConsoleWrapper";
 import { RenderComponent } from "../UserGuide/UserGuide";
-import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile"; //v2
+import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 import { checkAllCommandsAvailability } from "../../utils/CommandAvailability";
 import InstallationModal from "../InstallationModal/InstallationModal";
 
 /**
  * Represents the form values for the AirbaseNG component.
+ * @interface
  */
 interface FormValuesType {
     fakeHost: string;
     channel: string;
     macAddress: string;
     replayInterface: string;
     filePath: string;
+    verbosity: boolean; // Advanced mode: Verbosity flag
+    disableBeacons: boolean; // Advanced mode: Disable beacons
     customConfig: string;
+    customScript: string; // Custom mode: Custom script path
 }
 
 /**
- * The AirbaseNG component.
+ * The AirbaseNG component handles the setup and execution of the airbase-ng tool.
+ * Supports Basic, Advanced, and Custom modes for flexibility in input parameters.
  * @returns The AirbaseNG component.
  */
 const AirbaseNG = () => {
-    // Component State Variables.
-    const [loading, setLoading] = useState(false); // State variable to indicate loading state.
-    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
-    const [pid, setPid] = useState(""); // State variable to store the process ID of the command execution.
-    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    // State variables to manage the component's functionality.
+    const [loading, setLoading] = useState(false);
+    const [output, setOutput] = useState("");
+    const [pid, setPid] = useState("");
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false);
     const [allowSave, setAllowSave] = useState(false);
     const [hasSaved, setHasSaved] = useState(false);
-    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
-    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+    const [opened, setOpened] = useState(!isCommandAvailable);
+    const [loadingModal, setLoadingModal] = useState(true);
```

Comment:
Leave original comment. 
```suggestion
    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
```